src/ncp/CMakeLists.txt

```diff
@@ -91,7 +91,15 @@ target_include_directories(openthread-rcp PUBLIC ${OT_PUBLIC_INCLUDES} PRIVATE $
 
 target_sources(openthread-ncp-ftd PRIVATE ${COMMON_SOURCES})
 target_sources(openthread-ncp-mtd PRIVATE ${COMMON_SOURCES})
-target_sources(openthread-rcp PRIVATE ${COMMON_SOURCES})
+
+target_sources(openthread-rcp PRIVATE
+    changed_props_set.cpp
+    ncp_base.cpp
+    ncp_base_radio.cpp
+    ncp_base_dispatcher.cpp
```

Comment:
Sort source files?

Reply:
pushed new commit on top to also sort the list for ftd/mtd builds